app/src/components/publish/components/AttachmentsFileCard.tsx

```diff
@@ -1,59 +1,61 @@
 import { mdiFileOutline } from '@mdi/js';
 import Icon from '@mdi/react';
-import { Theme } from '@mui/material';
 import Box from '@mui/material/Box';
 import Chip from '@mui/material/Chip';
 import Paper from '@mui/material/Paper';
-import { makeStyles } from '@mui/styles';
-import clsx from 'clsx';
+import useTheme from '@mui/material/styles/useTheme';
 import { PublishStatus } from 'constants/attachments';
 
-const useStyles = makeStyles((theme: Theme) => ({
-  importFile: {
-    display: 'flex',
-    paddingTop: theme.spacing(2),
-    paddingBottom: theme.spacing(2),
-    paddingRight: theme.spacing(2),
-    paddingLeft: '20px',
-    overflow: 'hidden',
-    '& .importFile-icon': {
-      color: '#1a5a96'
-    },
-    '&.error': {
-      borderColor: '#ebccd1',
+const useStyles = () => {
+  const theme = useTheme();
+
+  return {
+    importFile: {
+      display: 'flex',
+      paddingTop: theme.spacing(2),
+      paddingBottom: theme.spacing(2),
+      paddingRight: theme.spacing(2),
+      paddingLeft: '20px',
+      overflow: 'hidden',
       '& .importFile-icon': {
-        color: theme.palette.error.main
+        color: '#1a5a96'
```

Comment:
Should this still use the theme.palette.error.main?

Reply:
Looks like the diff is just confused. None of the contents of the classes was changed, they just got pushed down a few lines because the wrapping function changed.